.github/workflows/notify-release.yml

```diff
@@ -9,5 +9,3 @@ jobs:
     steps:
       - name: Notify release
         uses: nearform/github-action-notify-release@v1
-        with:
```

Comment:
I've removed this line since the *github-action-notify-release* doesn't need the `github-token` anymore.